py4DSTEM/preprocess/relativity.py

```diff
@@ -16,8 +16,8 @@
 import matplotlib.pyplot as plt
 from ipywidgets import FloatProgress
 from time import time
-from ..utils import get_shift
-from ...io import DataCube
+from ..process.utils import get_shift
+from ..io.datastructure.py4dstem import DataCube
```

Comment:
I think you should always import `py4DSTEM.io.DataCube`. We've had problems before where multiple instances of the class, with different `id`, get created due to importing through different paths. When this happens, `assert isinstance(datacube, py4DSTEM.io.DataCube` can fail